From 612644ccd045a5b895e7fb3c56de0c426327233d Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
Date: Mon, 8 Aug 2016 15:52:36 +0200
Subject: [PATCH] Fix build of clutter tests

Various clutter test directly use cogl symbols, so they should be linked
against mutter-cogl as well.

Signed-off-by: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
---
 clutter/tests/interactive/Makefile.am | 4 +++-
 clutter/tests/micro-bench/Makefile.am | 5 ++++-
 2 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/clutter/tests/interactive/Makefile.am b/clutter/tests/interactive/Makefile.am
index ebea6d2..24a7f48 100644
--- a/clutter/tests/interactive/Makefile.am
+++ b/clutter/tests/interactive/Makefile.am
@@ -109,7 +109,9 @@ clean-wrappers:
 
 .PHONY: wrappers clean-wrappers
 
-common_ldadd = $(top_builddir)/clutter/libmutter-clutter-@CLUTTER_API_VERSION@.la
+common_ldadd = \
+  $(top_builddir)/clutter/libmutter-clutter-@CLUTTER_API_VERSION@.la \
+  $(top_builddir)/../cogl/cogl/libmutter-cogl.la
 
 check_PROGRAMS = test-interactive
 check_SCRIPTS = wrappers
diff --git a/clutter/tests/micro-bench/Makefile.am b/clutter/tests/micro-bench/Makefile.am
index 9060c38..2731b9e 100644
--- a/clutter/tests/micro-bench/Makefile.am
+++ b/clutter/tests/micro-bench/Makefile.am
@@ -1,4 +1,7 @@
-common_ldadd = $(top_builddir)/clutter/libmutter-clutter-@CLUTTER_API_VERSION@.la
+common_ldadd = \
+  $(top_builddir)/clutter/libmutter-clutter-@CLUTTER_API_VERSION@.la \
+  $(top_builddir)/../cogl/cogl/libmutter-cogl.la
+
 
 check_PROGRAMS = \
 	test-text \
-- 
2.8.1

